### ISSUE TITLE
zig-uri breaking update

### ISSUE THREAD

**Issue by nektro** [truemedian/zfetch#10]
https://github.com/MasterQ32/zig-uri/commit/e879df3a236869f92298fbe2db3c25e6e84cfd4c

```
./.zigmod/deps/git/github.com/truemedian/zfetch/src/request.zig:217:69: error: expected optional type, found '[]const u8'
        try self.client.writeStatusLineParts(method.name(), self.uri.path orelse "/", self.uri.query, self.uri.fragment);
                                                                    ^
```

**Comment by truemedian** [comment]
I can't make this change until that commit gets published on gyro. I'll give mattnite a bit of time for that, if it ends up not happening I'll switch gyro over to track the git repo and then fix this. This isn't a problem for gyro or submodules yet.

**Comment by truemedian** [comment]
Fixed with 88cc9de
